src/NzbDrone.Core/Download/Clients/Porla/README.md

```diff
@@ -0,0 +1,63 @@
+# Porla Client
```

Comment:
Sections of this would be better as comments in the code, other pieces could be dropped.

Reply:
I'll figure out what to do with this at the end.